lib/synapse.rb

```diff
@@ -33,6 +34,8 @@ def initialize(opts={})
       # configuration is initially enabled to configure on first loop
       @config_updated = true
 
+      @rate_limiter = RateLimiter.new(opts['rate_limiter_path'])
```

Comment:
Think we should log something if this path doesn't exist? If I understand correctly that would mean that restarting synapse would restart haproxy every time?